src/settingsmanager.cpp

```diff
@@ -89,11 +89,18 @@ void SettingsManager::setMonitorDir(QString monitorDir)
     emit(monitorDirChanged(monitorDir));
 }
 
+void SettingsManager::toggleTrashCheck()
```

Comment:
`toggleTrashCheck` is really implementation oriented.
Functions should be named about the "functionnal" they do, not how they are implemented.

A better name should be `toggleMoveToTrash`